std-bits/table/src/main/java/org/enso/table/data/column/operation/map/BinaryMapOperation.java

```diff
@@ -27,7 +27,7 @@ public MapOperation(String name) {
    * @param problemBuilder the builder allowing to report computation problems
    * @return the result of running the operation
    */
-  public abstract Storage<?> runMap(
+  public abstract Storage<?> runBiMap(
```

Comment:
Thought generally we avoided truncated names so should probably be `runBinaryMap`.

Reply:
Fair!

Reply:
I was keeping the original nomenclature (we used to have `bimap` here), but I think you are right, we should try to be consistent, I will rename.